src/main/java/Duke.java

```diff
@@ -1,10 +1,105 @@
+import java.util.Scanner;
+import Task.Deadline;
+import Task.ToDo;
+import Task.Events;
+import Task.Task;
+
 public class Duke {
+
+    static Task[] t = new Task[100];
```

Comment:
Hi, 
instead of using 100 which is a magic number, you can give meaning by defining it as a constant.

Reply:
To add on, i suggest you rename your objectname to something more descriptive/readable such as "task"